### ISSUE TITLE
Problem when auth is disabled in Uptime Kuma (using SSO instead)

### ISSUE THREAD

**Issue by openstep** [lucasheld/uptime-kuma-api#4]
We are running Uptime Kuma in a container and have SSO in front of it. Therefore we would like to disable auth in Uptime Kuma.
Here I encountered an error
```
>>> api.login('', '')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/A69778720/csaba/itsh/_programming/work/uptime-kuma-api/uptime_kuma_api/api.py", line 931, in login
    return self._call('login', {
  File "/Users/A69778720/csaba/itsh/_programming/work/uptime-kuma-api/uptime_kuma_api/api.py", line 320, in _call
    raise UptimeKumaException(r["msg"])
uptime_kuma_api.exceptions.UptimeKumaException: Incorrect username or password.
```

Do you have any plans to tackle this situation?

**Comment by lucasheld** [comment]
Thank you for creating this issue. This is implemented in version 0.3.0. The login method must be called without parameters when disableAuth is enabled.

```python
>>> from uptime_kuma_api import UptimeKumaApi
>>> api = UptimeKumaApi("http://127.0.0.1:3001")
>>> api.login()
>>> api.get_settings()
{'checkUpdate': True, 'searchEngineIndex': False, 'entryPage': 'dashboard', 'keepDataPeriodDays': 180, 'tlsExpiryNotifyDays': [7, 14, 21], 'trustProxy': False, 'disableAuth': True}
```

**Comment by openstep** [comment]
Many thanks for the reply and for the information.
